src/pudl/extract/xbrl.py

```diff
@@ -85,7 +85,10 @@ def _get_sqlite_engine(
         f"Dropping the old FERC Form {form_number} XBRL derived SQLite DB if it exists."
     )
     db_path = output_path / f"ferc{form_number}_xbrl.sqlite"
+
+    logger.info(f"Connecting to SQLite at {db_path}...")
```

Comment:
This logline seems useful enough to keep around - handy to know which DB file is failing.